src/user/redux/userActions.ts

```diff
@@ -0,0 +1,10 @@
+export const USER_ACTION_TYPE = {
+  LOGIN: 'user/LOGGED_IN',
+};
+
+export const userAction = {
+  login: (token: string) => ({
+    type: USER_ACTION_TYPE.LOGIN,
+    payload: { token },
+  }),
+};
```

Comment:
This piece of code seemed to have been left unused. There would probably be zero use of keeping it at the project at the moment.

Reply:
Thanks! Removed